Src/Sarif.PatternMatcher.Sdk/ExtensionMethods.cs

```diff
@@ -1,10 +1,42 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+
 namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Sdk
 {
     public static class ExtensionMethods
     {
+        public static HttpResponseMessage ResponseHeadersRead(this HttpClient httpClient, HttpRequestMessage request)
```

Comment:
Is is called `ReponseHeadersRead` because that's the operation specified in SendAsync and as a rule I tend to flow framework names around. `ReadResponseHeaders` is a clearer name, though, so I switched to that. Better?